pkg/proxy/proxy.go

```diff
@@ -145,6 +145,110 @@ func (p *SprayProxy) HandleProxy(c *gin.Context) {
 	c.String(http.StatusOK, "proxied")
 }
 
+func (p *SprayProxy) HandleProxyEndpoint(c *gin.Context) {
+	// currently not distinguishing between requests we can parse and those we cannot parse
+	metrics.IncInboundCount()
```

Comment:
Okay for now. We may decide later on that we need labels on this metric to distinguish the endpoints.